src/open_inwoner/utils/decorators.py

```diff
@@ -128,17 +129,19 @@ def wrapped(*args, **kwargs) -> RT:
                         raise
 
             cache_key = cache_key_with_attr_placeholders.format(**key_kwargs)
-            logger.debug("Resolved cache_key `%s` to `%s`", key, cache_key)
+            logger.debug(
+                "Resolved cache_key", original_key=str(key), cache_key=str(cache_key)
```

Comment:
I think both are guaranteed to be strings:

```suggestion
                "Resolved cache_key", original_key=key, cache_key=cache_key
```